data/aliases.js

```diff
@@ -587,6 +588,8 @@ exports.BattleAliases = {
 	"mechatyranitar2": "MT2",
 	"monica": "Giant",
 	"majin": "Spirit",
+	"gross": "Metagross",
```

Comment:
Yeah it's alphabetical I know, but I wasn't completely sure if putting them there was correct.